Spec/Utilities.swift

```diff
@@ -201,7 +201,6 @@ class Utilities: QuickSpec {
             }
 
             context("EventEmitter") {
-                
 
                 var eventEmitter = ARTInternalEventEmitter<NSString, AnyObject>(queue: AblyTests.queue)
```

Comment:
The `EventEmitter` instance is using the `AblyTests.queue` so `delay` is not ideal because it uses the main queue.